### PR TITLE
[chore] Bump version to 3.3.0

### PR DESCRIPTION
**Pull request by lydiasama**
## What happened 👀

Bump version to 3.3.0 for development

### CODE REVIEW

File: RxJavaTemplate/buildSrc/src/main/java/Versions.kt [hunk 1]
```diff
@@ -1,60 +1,60 @@
 object Versions {
-     const val BUILD_GRADLE_VERSION = "4.2.2"
```
Comment (doannimble) [inline RxJavaTemplate/buildSrc/src/main/java/Versions.kt:2]: @lydiasama Is it an unrelated change 😄  ?
Same for others.

Comment (lydiasama) [reply RxJavaTemplate/buildSrc/src/main/java/Versions.kt:2]: It's from my AS formatting automatically. 😱

Comment (luongvo) [reply RxJavaTemplate/buildSrc/src/main/java/Versions.kt:2]: @hoangnguyen92dn why did we use 5 spaces to tab in your last PR?

Comment (lydiasama) [reply RxJavaTemplate/buildSrc/src/main/java/Versions.kt:2]: I think my format is correct, we use 4 spaces instead of 5. @doannimble

**Review by luongvo — APPROVED** [review]
lgtm

**Review by chornerman — APPROVED** [review]

File: RxJavaTemplate/buildSrc/src/main/java/Versions.kt (hunk 1 above)
Comment (doannimble) [reply RxJavaTemplate/buildSrc/src/main/java/Versions.kt:2]: @lydiasama Yeah, I know your format is correct and comes from AS.
As I used to be warned it when pushing code with auto-format code from 7-11 project, then I just wanna make a confirmation about this one :D.

**Review by doannimble — APPROVED** [review]

File: RxJavaTemplate/buildSrc/src/main/java/Versions.kt (hunk 1 above)
Comment (lydiasama) [reply RxJavaTemplate/buildSrc/src/main/java/Versions.kt:2]: @doannimble Ok. Then I'll generate another ticket to format this file. It's just a format imo. 😌

Comment (doannimble) [reply RxJavaTemplate/buildSrc/src/main/java/Versions.kt:2]: @lydiasama Totaly agree with you, it does not a matter or a big thing. But it would be great if we can make our process consistent even it looks a bit cumbersome 🙏

**Comment by lydiasama** [comment]
@doannimble Done. 🙆🏻‍♀️

**Comment by lydiasama** [comment]
@doannimble Can I have your approval again?

**Review by doannimble — APPROVED** [review]
